### DISCUSSION TITLE
Discord Provider Profile: Set variables

### DISCUSSION THREAD

**Discussion by ustokan** [nextauthjs/next-auth#939] [Help] [answered]
**Your question**
How to set custom variables

**What are you trying to do**
Set custom variables in Discord provider profile
![image](https://user-images.githubusercontent.com/55207603/101662209-468c4e80-3a5a-11eb-8acb-3cee7c6f4cd2.png)
![image](https://user-images.githubusercontent.com/55207603/101662264-560b9780-3a5a-11eb-8691-73b38dcc57c8.png)

**Comment by balazsorban44** [comment, marked as answer]
Think it's related/duplicate. Please check the docs! 
https://github.com/nextauthjs/next-auth/discussions/937#discussioncomment-179677
